Applications/Utils/MeshEdit/UnityPreprocessing.cpp

```diff
@@ -0,0 +1,245 @@
+/*
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include <string>
+#include <vector>
+
+#include <tclap/CmdLine.h>
+
+#include "Applications/ApplicationsLib/LogogSetup.h"
+
+#include "MeshLib/Mesh.h"
+#include "MeshLib/MeshInformation.h"
+#include "MeshLib/Elements/Line.h"
+#include "MeshLib/Elements/Tri.h"
+#include "MeshLib/Elements/Quad.h"
+#include "MeshLib/Elements/Tet.h"
+#include "MeshLib/Elements/Hex.h"
+#include "MeshLib/Elements/Pyramid.h"
+#include "MeshLib/Elements/Prism.h"
+#include "MeshLib/IO/readMeshFromFile.h"
+#include "MeshLib/IO/VtkIO/VtuInterface.h"
+#include "MeshLib/MeshSearch/ElementSearch.h"
+#include "MeshLib/MeshEditing/RemoveMeshComponents.h"
+
+template <class T>
+bool checkType(MeshLib::Properties const& props, std::string const& name)
+{
+    if (props.existsPropertyVector<T>(name))
+    {
+        MeshLib::PropertyVector<T> const*const vec = props.getPropertyVector<T>(name);
+        if (vec->getMeshItemType() == MeshLib::MeshItemType::Cell)
+            return true;
+    }
+    return false;
+}
+
+bool isCellVec(MeshLib::Properties const& props, std::string const& name)
```

Comment:
Some data types are missing: E.g., unsigned long, std::size_t, float.
Some of them might not be important.